x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -122,15 +137,57 @@ const EventDetailsComponent: React.FC<Props> = ({
     [alerts, browserFields, data, id, isAlert, timelineId, timelineTabType]
   );
 
-  const selectedTab = useMemo(() => tabs.find((t) => t.id === view) ?? tabs[0], [tabs, view]);
+  const selectedEventTab = useMemo(() => tabs.find((t) => t.id === eventView) ?? tabs[0], [
+    tabs,
+    eventView,
+  ]);
+
+  const isThreatPresent: boolean = useMemo(
+    () =>
+      selectedEventTab.id === tabs[0].id &&
+      isAlert &&
+      !!data.find((item) => item.field === 'threat.indicator'),
+    [tabs, selectedEventTab, isAlert, data]
+  );
+
+  const threatTabs: EuiTabbedContentTab[] = useMemo(() => {
+    const defaultProps = { data, eventId: id, timelineId };
+    return isAlert && isThreatPresent
+      ? [
+          getSummaryTab(EventsViewType.threatSummaryView, i18n.THREAT_SUMMARY, {
+            ...defaultProps,
+            isDisplayingThreatSummary: true,
```

Comment:
It looks like the Summary tab is not correctly displaying alerts with multiple matches: 

<kbd><img width="409" alt="Detections_-_Kibana" src="https://user-images.githubusercontent.com/657252/112911242-9bf87400-90ba-11eb-82a0-348725119116.png"></kbd>

Compare this to the table view, which does display them correctly:
<kbd>
<img width="726" alt="30058990-90e2-11eb-8c1f-71bc0bba3ed5_-_Kibana" src="https://user-images.githubusercontent.com/657252/112911470-1c1ed980-90bb-11eb-939c-67b6b81787e7.png">
</kbd>

This situation can occur if you have an indicator match rule with multiple mappings, and more than one matches.

Reply:
hmm it looks like this is a UI issue, displayed values are correct but are squished into a single word (for instance, the value for threat.indicator.matched.type is urlemail, while the table puts a newline between url and email)

I think it makes sense to separate the values with a `,` in this case, updating as such for now.

Reply:
I think the values need to be separated into separate draggable elements, as sending "value, othervalue" to the timeline would not result in a correct query.

Reply:
hey that's a great point, will update as such